src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,241 @@
+			checked={selectedComponent.hideTextInput}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					hideTextInput: checked
+				});
+			}} />
+
+		<Checkbox
+			labelText='Light'
+			id='light'
+			checked={selectedComponent.light}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					light: checked
+				});
+			}} />
+
+		<TextInput
+			value={selectedComponent.labelText}
+			labelText='Slider label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				labelText: event.currentTarget.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.min}
+			label='Min'
+			onChange={(event: any, data: any, value: any) => {
+                setComponent({
+                    ...selectedComponent,
+                    min: value ? value : data.value
+                });
+            }} />
+
+		<NumberInput
+			value={selectedComponent.max}
+			label='Max'
+			onChange={(event: any, data: any, value: any) => setComponent({
+				...selectedComponent,
+				max: value ? value : data.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.value}
+			label='Value'
+			onChange={(event: any, data: any, value: any) => setComponent({
+				...selectedComponent,
+				value: value ? value : data.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.step}
+			label='Step'
+			onChange={(event: any, data: any, value: any) => setComponent({
+				...selectedComponent,
+				step: value ? value : data.value
+			})} />
+
+		<NumberInput
+			value={selectedComponent.stepMultiplier}
+			label='Step multiplier'
+			onChange={(event: any, data: any, value: any) => setComponent({
+				...selectedComponent,
+				stepMultiplier: value ? value : data.value
+			})} />
+
+		<TextInput
+			value={selectedComponent.minLabel}
+			labelText='Min label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				minLabel: event.currentTarget.value
+			})} />
+
+		<TextInput
+			value={selectedComponent.maxLabel}
+			labelText='Max label'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				maxLabel: event.currentTarget.value
+			})} />
+	</>;
+};
+
+export const ASliderCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}}
+/>;
+
+export const ASlider = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			<Slider
+			id='slider'
+			labelText={componentObj.labelText}
+			min={componentObj.min}
+			max={componentObj.max}
+			value={componentObj.value}
+			disabled={componentObj.disabled}
+			step={componentObj.step}
+			hideTextInput={componentObj.hideTextInput}
```

Comment:
hideTextInput should be called differently because the name implies it's a function.
It's not really a problem for json model itself but that can eventually make its way to state and exported code which might make it confusing.